.ci/wercker.sh

```diff
@@ -430,6 +430,17 @@ no-warning-imports-java-design-patterns)
   fi
   ;;
 
+no-error-pmd)
```

Comment:
you need to add it to wercker.yml

to fix wercker issue, please rebase to latest master.

Reply:
Fixed